source/css/sass/patterns/organisms/content-header.scss

```diff
@@ -1,23 +1,19 @@
-@import "../../variables/baselinegrid";
-@import "../../variables/color";
-@import "../../variables/content_header";
 @import "../../functions/types";
 @import "../../mixins/decorations";
 @import "../../mixins/media-query";
 @import "../../mixins/spacing";
-@import "../../mixins/typography";
+@import "../../variables/baselinegrid";
+@import "../../variables/content_header";
 
 .content-header {
-  color: $color-text-normal;
   text-align: center;
-  @include blg-spacing("block-start", "medium", "margin");
-  @include blg-spacing("block-end", "small", "margin");
+  @include padding($baselinegrid-space-medium, "block-start");
+  @include block-spacing($baselinegrid-space-medium);
```

Comment:
`block-margin` sounds preferable rather than creating indirection, but maybe better to just use `margin(0 $baselinegrid-space-medium, block)`...

Reply:
Happy to use `margin(0 $baselinegrid-space-medium, block)` here and on [item-tags.scss line 6](https://github.com/libero/pattern-library/pull/66/files#diff-f6ceb6d05ada36d3e1c71fbc07627d45R6), and retain the `block-spacing` mixin name for use in most other situations (those without the exception of applying padding as well).

Reply:
Wonder whether it's useful to remove the extra level of `block-spacing`, and rename/reorder the existing mixins, so:

```scss
@include padding-block-start($baselinegrid-space-medium);
@include margin-block(0 $baselinegrid-space-medium);
```

or

```scss
@include padding(block-start, $baselinegrid-space-medium);
@include margin(block, 0 $baselinegrid-space-medium);
```

Helps change https://github.com/libero/pattern-library/blob/9bd45b3b16f8f1e41715a89079dad767dbc26ce0/source/css/sass/mixins/_grid.scss#L7-L9 to

```scss
@include margin(0 auto);
@include padding-inline($grid-column_gap);
```

which matches CSS more closely.

Reply:
I prefer the first suggestion because I like the idea of having a mixin / mixins that are named to indicate they have something to do with overall spacing approach. The intent being to communicate that the block spacing is a system and people shouldn't just be tweaking the margin / padding a pixel here and a pixel there just to get things to arbitrarily line up: they should be letting baseline grid handle that. That's why in eLife we have the (admittedly badly named) `blg-spacing` mixin and its relatives.